clcache.py

```diff
@@ -192,7 +192,7 @@ def clean(self, stats, maximumSize):
         for o in objects:
             try:
                 objectInfos.append((os.stat(o), o))
-            except WindowsError:
+            except OSError:
```

Comment:
Because on Ubuntu I always get `E:195,19: Undefined variable 'WindowsError' (undefined-variable)` and a non-zero return value.

Since this is the only place where pylint reacts differently on both platforms and `WindowsError` is a `OSError`, it is just convenient to replace it.